src/Network/Network/ChangeLog.md

```diff
@@ -16,8 +16,10 @@
     ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
     * Overview of change #1
         - Additional information about change #1
---->
```

Comment:
Please return this closing tag back

Reply:
It should be --> ?

Reply:
@abjai should be `--->`

Reply:
@number213 

I checked ChangeLog.md for Compute and Monitor projects and its --> .  I remember that comments should be inside this <!-- comments --> block. Can you please confirm?

Adding ---> will make everything comments.

I also checked the same in VS and could see that --> is putting it in right manner.

Ref: https://github.com/Azure/azure-powershell/blame/network-may/src/Compute/Compute/ChangeLog.md

Reply:
You're right, `-->` also works. I wanted to avoid unnecessary changes to the file (since `--->` also works) but using correct html comment tag is also good.